README.md

```diff
@@ -71,20 +71,33 @@ Alternatively, you can also use an `onPositionChange` event to just get
 notified when the waypoint's position (e.g. inside the viewport, above or
 below) has changed.
 
-```javascript
+```jsx
 <Waypoint
   onPositionChange={this._handlePositionChange}
 />
 ```
 
+Waypoints can take a child, allowing you to track when a section of content
```

Comment:
Yeah what you have is probably fine here. If there's confusion, we can adjust the docs then.